index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css"> 
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1 id="title">Survey-Form</h1>
+    <p id="description"></p>
```

Comment:
I notice you are missing text in the `<p>` tag :thinking: ; try to write the text missing or delete the element if not needed :smiley:  to have a clean code :laughing: